### PR TITLE
Qe 14 socket

### PR DESCRIPTION
**Pull request by popovic65**

### CODE REVIEW

File: backend/src/controllers/auth.controller.ts [hunk 1]
```diff
@@ -28,6 +27,7 @@ export const signInWithGoogle = async (req: Request, res: Response) => {
         .cookie("token", token, {
           httpOnly: true,
           secure: process.env.NODE_ENV === "production",
+          expires: new Date(Date.now() + 24 * 60 * 60 * 1000),
```
Comment (ascepanovic) [inline backend/src/controllers/auth.controller.ts]: must be configurable as env variable -> amount of hours or minutes

File: backend/src/controllers/game.controller.ts [hunk 2]
```diff
@@ -15,8 +15,7 @@ export const findAllGamesController = async (
   res: Response
 ): Promise<void> => {
   try {
-    const { gameId } = req.body;
-    const games = await getPlayersInGame(gameId);
+    const games = await findAll();
```
Comment (ascepanovic) [inline backend/src/controllers/game.controller.ts]: all ? not smart

File: backend/src/routes/category.route.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ router.post("/", createCategoryController);
 
 /**
  * @swagger
- * /category/{id}:
+ * /api/category/{id}:
```
Comment (ascepanovic) [inline backend/src/routes/category.route.ts:75]: LIKE it, good!

File: backend/src/services/user.service.ts [hunk 4]
```diff
@@ -73,3 +75,47 @@ export async function deleteUser(id: number) {
...
+  } else {
+    throw new Error("User not found");
+  }
+}
+export async function checkUserGameStatus(userId: number) {
+  const game = await prisma.games.findFirst({
+    where: {
+      OR: [
+        {
+          game_status: "WAITING",
```
Comment (ascepanovic) [inline backend/src/services/user.service.ts]: do not use strings hardcoded here, call it somehow

**Review by ascepanovic — CHANGES_REQUESTED** [review]

File: backend/src/controllers/auth.controller.ts (hunk 1 above)
Comment (ascepanovic) [reply backend/src/controllers/auth.controller.ts]: or even as db variable within table app_configuration

File: backend/src/services/user.service.ts (hunk 4 above)
Comment (ascepanovic) [reply backend/src/services/user.service.ts]: somehow as constant from somewhere -> pseudo: **GameStatus.WAITING**

File: backend/src/controllers/game.controller.ts (hunk 2 above)
Comment (popovic65) [reply backend/src/controllers/game.controller.ts]: Kako treba da bude? Ovu rutu bi admin koristio

Comment (ascepanovic) [reply backend/src/controllers/game.controller.ts]: @popovic65 moraces da imas neki limit ili paging, all ce vratiti sve rezultate od jednom, bez obzira sto je admin, i sto sada imamo jednu igru, takva prakasa u BE-u nije dobra, da vratis sve, gotovo nikad osim ako znas da se novi inserti ne desavaju a cak i tada stavljas limit.

**Review by ascepanovic — APPROVED** [review]

**Pull request merged**